tests/features/content_types/content_type_administration.feature

```diff
@@ -0,0 +1,48 @@
+@api
+Feature: Content type administration features
+  In order to easily structure the content of the European Commission
+  As an administrator
+  I want to be able to create, customize and delete content types
+
+  Background:
+    Given I am logged in as a user with the 'administrator' role
+
+  Scenario: As administrator, I create a custom content type translated through
+    entity translation, using workbench moderation and having 2 fields "Body" and
+    "Select option". I should be able to create a content with it
+    When I go to "admin/structure/types/add"
+    And I fill in "Name" with "Groovy type"
+    And I fill in "Machine-readable name" with "groovy_type"
```

Comment:
I tried several ways to do it with @javascript and that failed all the time. Taking into account the time on it, I have chosen this way on purpose.